Fix #785: add squash support (#1366)
fabric8io/docker-maven-plugin: src/main/asciidoc/inc/external/_property_configuration.adoc

| *docker.noCache*
| Don't use a cache when building the image

| *docker.squash*

> | Squash newly built layers into a single new layer (API 1.25+, need to enable it in Daemon configuration file)

Reviewer: ```suggestion
| Squash newly built layers into a single layer (API 1.25+, needs to be enabled in the Docker daemon configuration)
```

Author: updated.
